science/liblsl/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+license             https://github.com/sccn/liblsl/blob/master/LICENSE
```

This is an **invalid** value for license.

The allowed values should be an identifier that identifies which kind of license this software has. You can see example values here: https://trac.macports.org/wiki/PortfileRecipes#licensekeyword

Taking a look, it looks like this should be the MIT license:

```suggestion
license             MIT
```

---

Thanks! I've updated the license.